wallet/src/de/schildbach/wallet/ui/dashpay/DashPayViewModel.kt

```diff
@@ -109,16 +112,12 @@ class DashPayViewModel(application: Application) : AndroidViewModel(application)
         emit(platformRepo.isPlatformAvailable())
     }
 
-    fun getNotificationCount(date: Long) {
-        notificationCountLiveData.value = date
+    fun searchNotifications(text: String) {
+        notificationsLiveData.searchNotifications(text)
     }
 
-    val getNotificationCountLiveData = Transformations.switchMap(notificationCountLiveData) { date: Long ->
-        liveData(Dispatchers.IO) {
-            val count = platformRepo.getNotificationCount(date)
-            if (count >= 0)
-                emit(count)
-        }
+    fun getNotificationCount(date: Long) {
```

Comment:
Are we going to use the `date` parameter in the future? If so, can we add a comment? Otherwise we should remove it.

Reply:
date was once used, now the associated liveData retrieves this from the preferences directly

https://github.com/dashevo/dash-wallet/pull/460/commits/929801ebf4d37deb29e65188d61588f5ddc7c7f1